lib/remote_storage/rest_provider.rb

```diff
@@ -506,5 +506,10 @@ def get_directory_listing_from_redis_via_lua(user, directory)
       items
     end
 
+    def validate_content_type(content_type)
```

Comment:
It's not really validating the content type itself, but merely the format of it.